### PR TITLE
added toast notifications

### PR DESCRIPTION
**Pull request by Ekwav**

### CODE REVIEW

File: public/serviceWorker.js [hunk 1]
```diff
@@ -89,15 +89,29 @@ firebase.initializeApp(firebaseConfig);
...
 messaging.onBackgroundMessage(function(payload) {
-    const notificationTitle = payload.notification.title;
-    const notificationOptions = {
-        body: payload.notification.body,
-        icon: '/logo192.png',
-        data: {
-            url: payload.data ? payload.data.onClick : "https://sky.coflnet.com"
-        }, //the url which we gonna use later
+    const request = indexedDB.open('keyval-store', 1);
+    var db;
```
Comment (matthias-luger) [inline public/serviceWorker.js:93]: use let (whole file)

File: public/serviceWorker.js [hunk 2]
```diff
@@ -89,15 +89,29 @@ firebase.initializeApp(firebaseConfig);
...
+        console.log("unkown notification type" + payload.data.type);
+        return;
+    }
+
+    var auction = JSON.parse(payload.data.auction);
+
+
+    var key = `auctionDetails\"${auction.uuid}\"`;
+    var data = {
+        expireTimeStamp: (new Date()).getTime() + 60 * 1000,
```
Comment (matthias-luger) [inline public/serviceWorker.js:104]: remove unnecessary brackets

File: src/utils/NotificationUtils.tsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
+import { toast } from "react-toastify";
```
Comment (matthias-luger) [inline src/utils/NotificationUtils.tsx:1]: format entiry file

File: src/utils/NotificationUtils.tsx [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+import cacheUtils from "./CacheUtils";
+
+export default function registerCallback(history)  {
+    var interval = setInterval(function() {
```
Comment (matthias-luger) [inline src/utils/NotificationUtils.tsx:5]: use let (everywhere)

**Review by matthias-luger — CHANGES_REQUESTED** [review]

File: public/serviceWorker.js (hunk 2 above)
Comment (Ekwav) [reply public/serviceWorker.js:104]: wich brackets are unnecessary here?

Comment (Ekwav) [reply public/serviceWorker.js:104]: Its the same as here: https://github.com/matthias-luger/hypixel-react/blob/master/src/utils/CacheUtils.tsx#L24

**Pull request merged**